### PR TITLE
style: update experiment selection label [WEB-1510]

### PR DESCRIPTION
**Pull request by hkang1**
## Description

No selection
Screenshot 2023-07-28 at 12 59 47 PM

Partial selection
Screenshot 2023-07-28 at 12 59 35 PM

All selection
Screenshot 2023-07-28 at 12 59 40 PM

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket
WEB-1510

### CODE REVIEW

**Comment by alexjdetermined** [comment]
Bugs
* "selected" label still showing when nothing is selected
* 
image

**Review by alexjdetermined — APPROVED** [review]
LGTM

File: webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx [hunk 1]
```diff
@@ -322,6 +322,21 @@ const TableActionBar: React.FC<Props> = ({
     }));
   }, [availableBatchActions]);
 
+  const selectionLabel = useMemo(() => {
+    let label = `${totalExperiments.toLocaleString()} experiment${totalExperiments > 1 && 's'}`;
```
Comment (EmilyBonar) [inline webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx:326]: nit: use `pluralizer` util

**Review by EmilyBonar — APPROVED** [review]
LGTM

File: webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx (hunk 1 above)
Comment (hkang1) [reply webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx:326]: niceeee

**Pull request merged**